### PR TITLE
Guidelines for creating commands

### PR DESCRIPTION
**Pull request by dennisseah**
Related to https://github.com/microsoft/bedrock/issues/1050

@andrebriggs, this is first draft. The team can improve on it.

### CODE REVIEW

File: guides/command-implementation.md [hunk 1]
```diff
@@ -26,3 +26,86 @@ have
...
    3. sample(s) of the command
+
+## Guidelines
+
+1. Consistent verb and option naming conventions. For example, we standardize on
+   `-o, --org-name` where `-o` is the alias and `--org-name` is the full option
+   name for organization option. We do not use another aliases other than `-o`
+   and other name other name like `--organization-name`. See
+   [Option Name Section](#option-name)
+2. Consistent command syntax. We use `commander` open source library to enforce
```
Comment (mtarng) [inline guides/command-implementation.md]: Can link to the repo for commander: https://github.com/tj/commander.js

File: guides/command-implementation.md [hunk 2]
```diff
@@ -26,3 +26,86 @@ have
    1. describe what this command is for (what it does)
    2. information that you think that users should be aware of
    3. sample(s) of the command
+
+## Guidelines
+
+1. Consistent verb and option naming conventions. For example, we standardize on
+   `-o, --org-name` where `-o` is the alias and `--org-name` is the full option
+   name for organization option. We do not use another aliases other than `-o`
+   and other name other name like `--organization-name`. See
```
Comment (mtarng) [inline guides/command-implementation.md]: "other name other name"

File: guides/command-implementation.md (hunk 1 above)
Comment (dennisseah) [reply guides/command-implementation.md]: accept

File: guides/command-implementation.md (hunk 2 above)
Comment (dennisseah) [reply guides/command-implementation.md]: accept

Comment (yradsmikham) [reply guides/command-implementation.md]: I think it should be "another alias"

**Review by yradsmikham — APPROVED** [review]

File: guides/command-implementation.md [hunk 3]
```diff
@@ -26,3 +26,86 @@ have
    1. describe what this command is for (what it does)
    2. information that you think that users should be aware of
    3. sample(s) of the command
+
+## Guidelines
```
Comment (sarath-p) [inline guides/command-implementation.md:30]: I would suggest adding a link to this doc in `contributing.md`.

**Review by sarath-p — APPROVED** [review]

File: guides/command-implementation.md (hunk 3 above)
Comment (dennisseah) [reply guides/command-implementation.md:30]: accept

**Review by andrebriggs — APPROVED** [review]
Looks great!

**Pull request merged**